beacon-chain/core/peerdas/helpers.go

```diff
@@ -234,15 +226,7 @@ func DataColumnSidecarsForReconstruct(
 
 		columnBytes := make([][]byte, 0, blobsCount)
 		for i := range column {
-			cell := column[i]
-
-			cellBytes := make([]byte, 0, kzg.BytesPerCell)
-			for _, fieldElement := range cell {
-				copiedElem := fieldElement
-				cellBytes = append(cellBytes, copiedElem[:]...)
-			}
-
-			columnBytes = append(columnBytes, cellBytes)
```

Comment:
These conversions are now not needed because we get the Cells as a flat sequence of bytes from p2p and we pass it to the kzg package as a flat sequence of bytes, the kzg package also returns it as a flat sequence of bytes